opensir/models/sir.py

```diff
@@ -40,6 +40,21 @@ class SIR(Model):
     IC = ["n_S0", "n_I0", "n_R0"]
     NAME = "SIR"
 
+    def predict(self, n_days=7):
+        """ Predict Susceptible, Infected and Recovered
+
+        Args:
+            n_days (int): number of days to predict
+
+        Returns:
+            np.array: Array with:
+                - T: current day
+                - S: Predicted number of susceptible
+                - I: Predicted number of infected
+                - R: Predicted number of recovered
+        """
```

Comment:
so, this means `predict` should always run after `fit`? So the user must run `solve` if there was no fit?

Reply:
Yes, that's what I think because it is very unlikely the case that the users have the exact parameters on a certain population prior prediction, particularly because the data is changing every day. This "forces" the user to fit prior predict, while she can still predict calling the function solve directly.

Alternatively, we could make a control flow statement so if the model was fitted, then `n_days = self.fit_attr{"t_obs"}[-1] + n_days.`, otherwise don't modify `n_days`.

Reply:
ok, I will do it as proposed then

Reply:
We had a great discussion on 12/05 and we will follow up on a call on 13/05 on what's exactly the input and output of the predict function :)